src/ts/settlement.ts

```diff
@@ -97,7 +114,7 @@ export const FULL_FEE_DISCOUNT = 10000;
  */
 export const MAX_TRADES_IN_SETTLEMENT = 2 ** 16 - 1;
 
-function encodeOrderFlags(flags: OrderFlags): number {
+function encodeTradeFlags(flags: TradeFlags): number {
```

Comment:
If we use the `uint8 flags` to encode the signing scheme as well, then maybe this could become:
```suggestion
function encodeSigningSchemeAndOrderFlags(scheme: SigningScheme, flags: OrderFlags): number {
```